p2p_wallet/Models/DerivableAccount.swift

```diff
@@ -8,12 +8,16 @@
 import Foundation
 import SolanaSwift
 
-struct DerivableAccount: Hashable {
+struct DerivableAccount: Hashable, Identifiable {
     let derivablePath: DerivablePath
     let info: KeyPair
     var amount: Double?
     var price: Double?
 
     // additional
     var isBlured: Bool?
+    
+    var id: String {
```

Comment:
just info.publicKey is enough, as each derivablePath has its own publicKey

Reply:
fixed